### DISCUSSION TITLE
Is there any sort of "level editor"?

### DISCUSSION THREAD

**Discussion by theresaway** [bjornbytes/lovr#562] [Q&A] [answered]
I'm really interested in this engine, but i'd rather not layout everything in my game scene all by code. Is there some sort of level editor or something?

**Comment by shakesoda** [comment, marked as answer]
lovr has no concept of scenes to begin with, so the simplest starting point for that is assembling a scene in blender and exporting it as gltf for lovr to load.

edit: next from there is something like [layout](https://github.com/sophiabaldonado/layout). haven't tried it myself, though.
